### DISCUSSION TITLE
dev build console error spam

### DISCUSSION THREAD

**Discussion by uBlock-user** [uBlockOrigin/uBlock-issues#2929] [General] [answered]
![Capture](https://github.com/uBlockOrigin/uBlock-issues/assets/21290713/b638eca6-022d-4fc9-be1d-59e878d31008)

Browser -  Nightly

**Comment by ghost** [comment]
![image](https://github.com/uBlockOrigin/uBlock-issues/assets/886325/350c7c8d-a7e3-46d4-af70-6dccd98e9344)

**Reply by uBlock-user** [reply]
Is that Firefox ? I only tested in Nightly.

**Reply by ghost** [reply]
Nightly. Timestamps to see how often it happens. Turns out some responses don't have type specified, for example on `https://www.windy.com/`.

**Reply by ghost** [reply]
Responses to `HEAD` requests?

**Reply by ghost** [reply]
![image](https://github.com/uBlockOrigin/uBlock-issues/assets/886325/a37005b9-3034-4b33-958c-dbf7736cdc6a)
![image](https://github.com/uBlockOrigin/uBlock-issues/assets/886325/368a2a1f-3754-4247-9ecc-5877f4f86b80)

**Comment by uBlock-user** [comment, marked as answer]
fixed in https://github.com/gorhill/uBlock/commit/4de6bd6f07efc6b06faa38ef7483f5faf379549a
